client/webserver/site/src/html/markets.tmpl

```diff
@@ -27,6 +27,7 @@
   <div class="d-flex flex-column flex-grow-1">
 
     {{- /* CHART */ -}}
+    <div class="fs15 pt-3 text-center d-hide errcolor" id="loaderErr"></div>
     <div id="marketChart" class="w-100">
```

Comment:
We might still be able to display the market info, so let's not put an error message here. Since we plan to support plugins for the server backends, and **core** is not the only DEX application, this isn't technically an error. Instead, hide the order forms and wallet info and put a message (not `errcolor`) where the order forms would be.